package.json

```diff
@@ -220,17 +220,24 @@
           "swift.path": {
             "type": "string",
             "default": "",
-            "markdownDescription": "Override the default path of the folder containing the Swift executables. The default is to look in the `PATH` environment variable. This path is also used to search for other executables used by the extension like `sourcekit-lsp` and `lldb`.",
-            "order": 1
+            "markdownDescription": "Override the default path of the folder containing the Swift executables. The default is to look in the `PATH` environment variable. This path is also used to search for other executables used by the extension like `sourcekit-lsp` and `lldb`."
           },
           "swift.buildArguments": {
             "type": "array",
             "default": [],
             "items": {
               "type": "string"
             },
-            "markdownDescription": "Additional arguments to pass to `swift build`. Keys and values should be provided as individual entries in the list. If you have created a copy of the build task in `tasks.json` then these build arguments will not be propogated to that task.",
-            "order": 2
+            "markdownDescription": "Additional arguments to pass to `swift build`. Keys and values should be provided as individual entries in the list. If you have created a copy of the build task in `tasks.json` then these build arguments will not be propogated to that task."
+          },
+          "swift.testBuildArguments": {
```

Comment:
So if `buildArguments` is always added maybe we call `testBuildArguments` something different. At first I thought it was in lieu of `buildArguments` which may cause some confusion. Maybe `additionalTestArguments`? It may then be worth cleaning up the markdown description for `buildArguments`, since it does come into play for more than just `swift build` commands, really it's any command that may build an artifact like `swift package describe`

Reply:
Good call, I've changed this.